src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js

```diff
@@ -65,14 +67,23 @@ export const ExistingSectionFolio = ({
     >
       <Row>
         <Col xs={12}>
-          <Selection
-            id="criterion-value-type"
-            value={dropdownValue}
-            dataOptions={existingRecordFields}
-            onChange={handleExistingRecordSelect}
-            onFilter={handleFieldSearch}
-            dirty={isDirty}
-          />
+          <FormattedMessage
+            id="ui-data-import.match.existing.record.field"
+            values={{ recordType: existingRecordLabel }}
+          >
```

Comment:
`existingRecordFieldLabel` prop is the same, consider converting this from `node` to `string` using `intl.formatMessage` method in the `MatchCriterion` component and use it here for `Section label` and `Selection aria-label`.

Reply:
done